Cargo.toml

```diff
@@ -20,15 +20,15 @@ serde_derive = "~1.0"
 rand = "~0.6"
 rand_chacha = "~0.1"
 regex = "~1.1"
-ring = "~0.14"
-recrypt = "~0.8"
+ring = { version= "~0.16", features = ["std"] }
+recrypt = "~0.8.1"
```

Comment:
This change is okay, but should have the same meaning as before.

Reply:
This is how @coltfred helped me fix the problem with the AesError

Reply:
That shouldn't have had any effect, but maybe there's something with caching in cargo that I don't understand.

Reply:
@clintfred The effect it has is that it ensures it will be at least 0.8.1 instead of at least 0.8.0. You're right that it doesn't really matter since it would always choose the latest anyways.

Also, Craig thought you were talking about the ring version line.